easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.1.95-GCC-10.2.0.eb

```diff
@@ -0,0 +1,65 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'Bundle'
+
+name = 'SentencePiece'
+version = '0.1.95'
```

Comment:
```suggestion
version = '0.1.96'
```

Reply:
@robert-mijakovic any thoughts on the suggestion for a minor version update?

Reply:
@verdurin Sorry for the delay. Done!